src/main/java/games/strategy/net/UniversalPlugAndPlayHelper.java

```diff
@@ -140,6 +140,7 @@ private String addPortForwardUPNP() {
     try {
       System.out.print(m_device.getExternalIPAddress());
     } catch (final UPNPResponseException | IOException e1) {
+      // ignore
```

Comment:
The original author's intent seemed to be to not display any kind of error here in the `Adding mapping from <fromHost>:<fromPort> To <toHost>:<toPort>` string and let it fail a few lines later where the user will see an appropriate error message.

Please advise if this should be changed to output something like `<FAILED TO OBTAIN EXTERNAL IP ADDRESS>` instead of nothing.